Sources/tart/DiskImageFormat.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import ArgumentParser
+
+enum DiskImageFormat: String, CaseIterable, Codable {
+  case raw = "raw"
+  case asif = "asif"
+
+  var displayName: String {
+    switch self {
+    case .raw:
+      return "RAW (UDIF read-write)"
```

Comment:
I think RAW is unrelated to UDIF:

```suggestion
      return "RAW"
```